server/views/partials/layout.njk

```diff
@@ -73,7 +73,7 @@
 {% endblock %}
 
 {% block beforeContent %}
-  {% if not noBackLink %}
+  {% if backLink !== false %}
```

Comment:
Hopefully an improvement on the double negative - need to check it is not false rather than true as not setting the variable as true globally so I can override as false in confirmation pages.